core/src/main/resources/lib/hudson/progressive-text.js

```diff
@@ -28,6 +28,12 @@ Behaviour.specify(
         parameters: { start: e.fetchedBytes },
         requestHeaders: headers,
         onComplete: function (rsp) {
+          if (rsp.status >= 500) {
+            setTimeout(function () {
+              fetchNext(e, href, onFinishEvent);
+            }, 1000);
+            return;
+          }
```

Comment:
```suggestion
          }
          if (rsp.status == 403) {
            // likely an expired crumb
            location.reload();
            return;
          }
```

Reply:
You mean for an expired servlet session?

Reply:
Yes, very basic setup. Basically I ran `mvn -pl war jetty:run`, opened a Pipeline log, then killed `mvn` and ran it again. When it came back up, I was greeted by a 403 error in the log due to the crumb.